src/components/sidePanel/SidePanel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './SidePanel.css'
+
+class SidePanel extends React.Component{
+  render(){
+    return(
+      <div className = "SidePanel">
+        <div style= {{marginTop:'50px'}}>
+          <a className = "homelink" href = "/">Sustainable Energy</a>
+        </div>
+
+        <div style={{marginTop:'100px'}}>
+          <p className = "navLinkSpacing"><i className="fas fa-book"></i><a  className = "navLinkStyle" href = "/literature">Literature Review</a></p>
+          
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
Class-based components are nearly a thing of the past. Time to hop on the functional component train, choo choo!
```suggestion
const SidePanel = () => (
  <div className = "SidePanel">
    <div style= {{marginTop:'50px'}}>
      <a className = "homelink" href = "/">Sustainable Energy</a>
    </div>

    <div style={{marginTop:'100px'}}>
      <p className = "navLinkSpacing">
        <i className="fas fa-book"></i><a  className = "navLinkStyle" href = "/literature">Literature Review</a>
      </p>          
    </div>
  </div>
);
```